CONTRIBUTNG.md

```diff
@@ -1,54 +0,0 @@
-# Contributing

```

Comment:
**Clarifying some jumbled text and first to third person phrasing** 
After a contributor has shown they're a good fit and have completed 1 or 2 issues labeled as "good first issue" with pull requests that have been accepted they can feel free to move on to other issues. We're always thrilled to see new contributors joining our community.